mocks/mocks.js

```diff
@@ -0,0 +1,294 @@
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "5 característica de um bom líder",
+    url: "https://catho.com.br",
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "O que falar e o que não falar em uma entrevista de emprego",
+    url: "https://catho.com.br",
+  },
+];
+
+const privateMockContent = [
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Design/UX",
+    url: "https://catho.com.br",
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Designer BH",
+    url: "https://catho.com.br",
+  },
+];
+
+const extraMockContent = [
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Vagas de home office",
+    url: "https://catho.com.br",
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "O que é home office?",
+    url: "https://catho.com.br",
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Como fazer uma carta de demissão?",
+    url: "https://catho.com.br",
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Como fazer um currículo sem experiência e sem ensino superior",
+    url: "https://catho.com.br",
+  },
+];
+
+const othersMockContent = [
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Novo Item",
+    url: "https://catho.com.br",
+  },
+  {
+    dataGtmEventAction: "novo-footer:parceiros-internacionais",
+    dataGtmEventCategory: "catho",
+    dataGtmEventLabel: "link:zhaopin",
+    dataGtmEventName: "interaction:normal-metric",
+    label: "Outros",
+    url: "https://catho.com.br",
+  },
+];
+
+const jobNotificationListMock = [
+  {
+    id: "123",
```

Comment:
In the console a warning about equal keys is shown because of this id that is the same between all elements in this list, I suggest putting different id in each of them